libs/platform/kernel/platform_kernel.vcxproj

```diff
@@ -28,7 +28,7 @@
     <ClCompile Include="..\ebpf_crypto_hash.c" />
     <ClCompile Include="..\ebpf_epoch.c" />
     <ClCompile Include="..\ebpf_error.c" />
-    <ClCompile Include="..\ebpf_etw.c" />
+    <ClCompile Include="..\ebpf_tracelog.c" />
```

Comment:
re-sort in alphabetical order

Reply:
`.vcxproj` files are managed by Visual Studio and are designed to be ordering-agnostic; also, they are not user-visible as source code.